data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -114,6 +114,16 @@ def iter_survey_answers(access_token: str, survey_id: str) -> Iterable[dict]:
         yield from response_json["data"]
 
 
+def parse_answers_part_in_survey_answers_response(question_response_json: dict):
+    result = {}
+    if "answers" in question_response_json:
+        for answer in question_response_json["answers"]:
+            for key in ["choice_id", "row_id", "col_id", "other_id", "text"]:
```

Comment:
I thought we have done this together before? Or was I just imagining it?

Reply:
Yes we did for the question part:)